public/static/partials/user_host_details.html

```diff
@@ -6,74 +6,60 @@ <h2 style="display:inline-block; padding-right:15px">
-      </button>
-      <ul class="dropdown-menu expire-dropdown" role="menu" aria-labelledby="extLen">
-        <li role="presentation" class="dropdown-header">Extension Durations</li>
-        <li role="presentation" ng-repeat="extensionLength in hostExtensionLengths">
-          <a role="menuitem" ng-click="setExtensionLength(extensionLength);">
-          [[extensionLength.display]]
-          </a>
-        </li>
-      </ul>
-    </span>
-    <span>
-      <button type="button" class="btn btn-info expire-button" style="float: right;" ng-click="updateHostExpiration()">
-      Update Host Expiration
-      </button>
-    </span>
+    <md-card style="width:99%">
+      <md-card-content>
+        <div class="entry">
+          <strong>ID</strong>
+          <span>[[curHostData.id]] </span>
+        </div>
+        <div class="entry">
+          <strong>Started at</strong> <span>[[curHostData.creation_time | convertDateToUserTimezone:userTz:"MMM D, YYYY h:mm:ss a"]]</span>
+        </div>
+        <div class="entry" ng-show="curHostData.isTerminated">
+          <strong>Terminated at</strong> <span>[[curHostData.termination_time | convertDateToUserTimezone:userTz:"MMM D, YYYY h:mm:ss a"]]</span>
+        </div>
+        <div class="entry">
+          <strong>Status</strong> <span>[[curHostData.status]]</span>
+        </div>
+        <div class="entry" ng-show="!curHostData.isTerminated">
+          <strong>SSH User</strong> <span>[[curHostData.user]]</span>
+        </div>
+        <div class="entry" ng-show="!curHostData.isTerminated">
+          <strong>DNS Name</strong> <span>[[curHostData.host]]</span>
+        </div>
+        <div class="entry" ng-show="!curHostData.isTerminated">
+          <strong>Work Directory</strong> <span class="mono">[[curHostData.distro.work_dir]]</span>
+        </div>
+        <div class ="entry" ng-show="curHostData.userdata">
+          <strong>User Data:</strong><br/>
+          <pre>[[curHostData.userdata]]</pre>
+        </div>
+        <div class="entry" ng-show="!curHostData.isTerminated && curHostData.host.length > 0 && (curHostData.distro.bootstrap_method != 'user-data' || curHostData.status == 'running')">
+          <strong>Login Command:</strong>
+          <pre readonly>ssh [[curHostData.user]]@[[curHostData.host]]</pre>
+        </div>
+      </md-card-content>
+
+    </md-card>
+      <section layout="row" flex ng-show="curHostData.isWinHost && curHostData.status == 'running'">
+        <div flex="85"></div>
+        <div class="entry" flex="15">
+          <md-button type="button" class="btn-info md-raised" style="top: 8px; float: right;" ng-click="openSpawnModal('updateRDPPassword')">
+            Set RDP Password
+          </md-button>
+        </div>
+      </section>
+      <section layout="row" flex style="white-space: nowrap">
+          <div flex="85">
+            <strong>Expires On:</strong>
+            <md-datepicker ng-model="date_for_expiration" md-placeholder="Date"></md-datepicker>
```

Comment:
Could you add the md-min-date property here to prevent setting dates in the past? I realize we're already validating on the server but this will make it more user-friendly